OneBusAway/ui/regional_alerts/RegionalAlertsViewController.swift

```diff
@@ -29,6 +29,7 @@ class RegionalAlertsViewController: OBAStaticTableViewController {
         self.hidesBottomBarWhenPushed = true
         let spacer = UIBarButtonItem.init(barButtonSystemItem: .flexibleSpace, target: nil, action: nil)
         let markReadButton = UIBarButtonItem.init(title:  NSLocalizedString("regional_alerts_controller.mark_all_as_read", comment: "Mark All as Read toolbar button title"), style: .plain, target: self, action: #selector(markAllAsRead))
+        markReadButton.setTitleTextAttributes([NSForegroundColorAttributeName: OBATheme.obaGreen], for: .normal)
```

Comment:
Instead of changing this one `UIBarButtonItem`, take a peek at `+[OBATheme setAppearanceProxies]`. I bet you can add a declaration for `UIBarButtonItem` there and have this change work now and forever.